ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -210,33 +229,70 @@ const Footer = () => {
     onCancel({ location: MetaMetricsEventLocation.Confirmation });
   }, [onCancel, shouldThrottleOrigin]);
 
+  const isShowShieldFooterCoverageIndicator = useEnableShieldCoverageChecks();
```

Comment:
Could we encapsulate this inside `ShieldFooterCoverageIndicator` so it's responsible for it's own render or not?